starport/templates/app/stargate/go.mod.plush

```diff
@@ -19,6 +19,104 @@ require (
 	google.golang.org/grpc v1.40.0
 )
 
+require (
```

Comment:
Can we remove this part @faddat for simplicity, it'll be added automatically by go mod anyway if needed?